pkg/scheduler/preemption/preemption.go

```diff
@@ -168,7 +179,23 @@ func (p *Preemptor) IssuePreemptions(ctx context.Context, preemptor *workload.In
 	workqueue.ParallelizeUntil(ctx, parallelPreemptions, len(targets), func(i int) {
 		target := targets[i]
 		if !meta.IsStatusConditionTrue(target.WorkloadInfo.Obj.Status.Conditions, kueue.WorkloadEvicted) {
-			message := preemptionMessage(preemptor.Obj, target.Reason)
+			preemptorPath := "/" + string(preemptor.ClusterQueue)
+			if snap.HasParent() {
+				ancestors := snap.PathParentToRoot()
+				for a := range ancestors {
+					preemptorPath = "/" + string(a.GetName()) + preemptorPath
+				}
+			}
+
+			preempteePath := "/" + string(target.WorkloadInfo.ClusterQueue)
+			if target.WorkloadCq != nil && target.WorkloadCq.HasParent() {
+				ancestors := target.WorkloadCq.PathParentToRoot()
+				for a := range ancestors {
+					preempteePath = "/" + string(a.GetName()) + preempteePath
+				}
+			}
+
```

Comment:
Please de-duplicate the logic, and use something like strings.Join(parts, "/") for readability